### PR TITLE
Update postgresql deployment documentation

### PR DESCRIPTION
**Pull request by pkliczewski**
This PR fixes spelling mistake in the command and add information about requirement of funcional storage class.

### CODE REVIEW

File: documentation/latest/gsg/ocp.md [hunk 1]
```diff
@@ -50,9 +50,13 @@ git clone {{ page.parodos.git_repo }} -b {{ page.parodos.git_branch }}
 
 ### Installing Parodos
 
+Before appling the manifests please make sure you have a strageclass configured
```
Comment (masayag) [inline documentation/latest/gsg/ocp.md:53]: s/strageclass/storageclass or even replace with:
`[storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/)`

File: documentation/latest/gsg/ocp.md [hunk 2]
```diff
@@ -50,9 +50,13 @@ git clone {{ page.parodos.git_repo }} -b {{ page.parodos.git_branch }}
 
 ### Installing Parodos
 
+Before appling the manifests please make sure you have a [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/)
+configured and set as default. It is needed to bind persistent volume
+claim used by postgresql.
+
```
Comment (gciavarrini) [inline documentation/latest/gsg/ocp.md:56]: Do you think we can provide a some suggestion/yaml snippet about the easiest way to configure it to let parodos work?

Comment (pkliczewski) [reply documentation/latest/gsg/ocp.md:56]: Good idea. What do you think about providing a link to https://docs.openshift.com/container-platform/4.13/post_installation_configuration/storage-configuration.html ?

**Comment by eloycoto** [comment]
TBH, I do not think that we should include the need to postgres in a GSG, I would avoid all to not create any miss-configuration at the user level.

**Comment by pkliczewski** [comment]
> TBH, I do not think that we should include the need to postgres in a GSG, I would avoid all to not create any miss-configuration at the user level.

What do you suggest for the users to use in their environments? Any persistence layer needs to be there.

**Comment by eloycoto** [comment]
As GSG, I would use an internal database as it is for kind, for example. You want to get started, not to complicated with external factors. IMHO

**Comment by pkliczewski** [comment]
@eloycoto I understand your point about getting started guide but we would need to have no recommendation how should be run long term. Internal db is not a solution if you want to survive container crash/restart/eviction.

**Comment by masayag** [comment]
@eloycoto @pkliczewski what's the verdict? can this be merged? :-)

**Comment by pkliczewski** [comment]
I think we have much bigger issues with our manifests than making sure to configure storage class. We need to start providing easy way to deploy known version of services. Document how to customize db and ldap settings and provide some getting started guide on how to deploy both. I am ok to close this PR.

**Pull request closed without merging**